Gemfile.lock

```diff
@@ -143,4 +143,4 @@ DEPENDENCIES
   sqlite3 (~> 1.4)
 
 BUNDLED WITH
-   2.4.10
+   2.4.10
```

Comment:
Same as above, maybe your editor is configured to remove the blank line, it would be a good idea to look into that and update your configurations.